bases/mundo_bm_wdi/country/table_config.yaml

```diff
@@ -0,0 +1,510 @@
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: ppp_survey_year
+      bigquery_type: string
+      description: Purchasing power parity survey year
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations: all values are originally NaN
+      is_in_staging: true
+      is_partition: false
+    - name: balance_of_payments_manual_in_use
+      bigquery_type: string
+      description: Type of balance of payments manual in use
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: external_debt_reporting_status
+      bigquery_type: string
+      description: Defines the status of external debt
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: system_of_trade
+      bigquery_type: string
+      description: Defines if the trade system is general or special
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: government_accounting_concept
+      bigquery_type: string
+      description: Defines if it's a consolidated or budgetary government
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: imf_data_dissemination_standard
+      bigquery_type: string
+      description: Types of data dissemination according to IMF
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: latest_population_census
+      bigquery_type: string
```

Comment:
se for ano, é inteiro

Reply:
o ano é acompanhado de um comentário como '2011. Population figures compiled from administrative registers.', uma opção seria obter só o ano em uma coluna e o texto em outra